features/headings.yml

```
@@ -0,0 +1,12 @@
+name: <h1> through <h6>
+description: The `<h1>` through `<h6>` HTML elements represent six levels of section headings, in order of decreasing importance.
+spec: https://html.spec.whatwg.org/multipage/sections.html#the-h1,-h2,-h3,-h4,-h5,-and-h6-elements
+group: html
```

```suggestion
group: html-elements
```

A new group just landed. So let's use it here.